radixdlt/src/main/java/com/radixdlt/consensus/VertexMetadata.java

```diff
@@ -129,6 +161,32 @@ public Hash getId() {
 		return id;
 	}
 
+	@JsonProperty("validator_set")
+	@DsonOutput(Output.ALL)
+	private Map<String, UInt256> getValidatorSetJson() {
+		if (validatorSet == null) {
+			return null;
+		}
+		return validatorSet.getValidators().stream()
+			.collect(ImmutableMap.toImmutableMap(v -> v.getNode().getKey().toBase64(), BFTValidator::getPower));

```

Comment:
We have pretty much exclusively used hex for pubkeys everywhere else.

If we want to change this, also needs changing in ECDSASignatures, TimestampedECDSASignatures and GenerateUniverses.

Reply:
fair enough